README.md

```diff
@@ -1,15 +1,37 @@
 # bfint
 
-BrainFuck interpreter implemented in Rust.
+This is a BrainFuck interpreter implemented in Rust. BrainFuck is an esoteric programming language known for its minimalism and extreme simplicity.
 
+## Usage
 
-## Roadmap
+The bfint-cli run command supports two options:
+
+- `[ARG]`: Positional an argument with BrainFuck code.
+- `--file <FILE>`: Specifies the file containing the BrainFuck code to run.
+
+Running BrainFuck code with positional argument:
+```bash
+bfint-cli run --<-<<+[+[<+>--->->->-<<<]>]<<--.<++++++.<<-..<<.<+.>>.>>.<<<.+++.>>.>>-.<<<+.
+```
+
+Running BrainFuck code from a file:
+```bash
+bfint-cli run --file
```

Comment:
for what "run" argument )
maybe make without it ?

Reply:
okay lets do it easier